DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/annotated_class_clean_coding/src/annotated_class_util.hpp

```diff
@@ -7,15 +7,17 @@ using namespace sycl::ext::oneapi::experimental;
 
 namespace fpga_tools {
 ///////////////////////////////////////////////////////////////////////////////
+//
 // This header provides a utility function: "alloc_annotated", which improves
 // "malloc_shared" in allocating host/share memory in the following aspects:
 //   (1) better code brevity
 //   (2) support compile-time type check
 //
 // "annotated_alloc" function takes an annotated_arg type as the only template
```

Comment:
```suggestion
// "alloc_annotated" function takes an annotated_arg type as the only template
```
you missed at least one... please use VS Code's search feature to look for any annotated_alloc still hanging around in your comments

Reply:
Searched in the code sample and all the occurrences are fixed.